mailpoet/assets/js/src/automation/editor/components/header/document_actions.tsx

```diff
@@ -68,7 +68,7 @@ export function DocumentActions({ children }): JSX.Element {
                     as="h1"
                   >
                     <VisuallyHidden as="span">
-                      {__('Editing workflow: ', 'mailpoet')}
+                      {__('Editing workflow:', 'mailpoet')}
```

Comment:
Should we not be referring to it as an automation in user-facing text like this?